File: crates/deslop/tests/fixtures/fused-golden-go/rename_lean_a.go
package golden

func CalibrateSensorDrift(readings []int, gainFactor int) int {
	driftSum := 0
	for _, readingValue := range readings {
		driftSum = driftSum + readingValue
	}
	gainAdjusted := driftSum * gainFactor
	driftScore := driftSum + gainAdjusted
	return driftScore
}

File: crates/deslop/tests/fixtures/fused-golden-csharp/rename_lean_a.cs
namespace Golden.RenameLean;

public static class SensorCalibrator
{
    public static long CalibrateSensorDrift(long[] readings, long gainFactor)
    {
        long driftSum = 0;
        foreach (long readingValue in readings)
        {
            driftSum = driftSum + readingValue;
        }

        long gainAdjusted = driftSum * gainFactor;
        long driftScore = driftSum + gainAdjusted;
        return driftScore;
    }
}
